Sources/MessagesDisplayDataSource.swift

```diff
@@ -26,7 +26,9 @@ import Foundation
 
 public protocol MessagesDisplayDataSource: class, MessagesDataSource {
 
-    func messageColorFor(_ message: MessageType, at indexPath: IndexPath, in messagesCollectionView: MessagesCollectionView) -> UIColor
+    func textColorFor(_ message: MessageType) -> UIColor
+    
+    func backgroundColorFor(_ message: MessageType) -> UIColor
```

Comment:
I feel like there could be a method for this information that way you have the option to get it or not. I was just finding the information redundant and that's why I chose that route.  But if you can see a good case for it I could add it back in.